src/Nelmio/Alice/Instances/Processor/Methods/ArrayValue.php

```diff
@@ -44,6 +44,7 @@ public function canProcess(ProcessableInterface $processable)
      */
     public function process(ProcessableInterface $processable, array $variables)
     {
+        /** @var array $values */
         $values = $processable->getValue();
```

Comment:
Such venom.

I'm not particularly worried about a lack of clarity here, but I'm not opposed to it either.